src/uid2ApiClient.ts

```diff
@@ -133,7 +135,7 @@ export class Uid2ApiClient {
     this._requestsInFlight.push(req);
     req.overrideMimeType('text/plain');
     req.open('POST', url, true);
-    req.setRequestHeader('X-UID2-Client-Version', this._clientVersion);
+    req.setRequestHeader('X-UID2-Client-Version', this._clientVersion); // TODO: EUID
```

Comment:
I meant to include the TODO - it still needs to be done and I'm not looking into it with this PR.